curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
+3. BitSet
+4. Bitmasks (Basic bit manipulation)
```

Comment:
- turn_on, turn_off, ... (mention them all)

Reply:
They are explained in CP in details